sample/host/src/AuthenticatedLayout.tsx

```diff
@@ -1,10 +1,46 @@
 import { Link, Outlet, useNavigate } from "react-router-dom";
-import type { RenderItemFunction, RenderSectionFunction } from "@squide/react-router";
-import { Suspense, useCallback } from "react";
-import { useEventBusListener, useNavigationItems, useRenderedNavigationItems } from "@squide/react-router";
+import { type ReactNode, Suspense, useCallback } from "react";
+import { useEventBusListener, useNavigationItems, useRenderedNavigationItems, type RenderItemFunction, type RenderSectionFunction, isNavigationLink, type NavigationLinkRenderProps, type NavigationSectionRenderProps } from "@squide/react-router";
 
 import { sessionManager } from "./session.ts";
 
+type RenderLinkItemFunction = (item: NavigationLinkRenderProps, index: number, level: number) => ReactNode;
+
+type RenderSectionItemFunction = (item: NavigationSectionRenderProps, index: number, level: number) => ReactNode;
```

Comment:
RenderMenuItemFunction?

Reply:
I removed "menu" from everywhere as it's too specific and it's similar to "section" which is a more genral term.

Reply:
What's off is seeing this however: 
const renderMenu: RenderSectionItemFunction

This interface is created specifically for this method, i feel like they should have a similar name

Reply:
Fixed the name of the render functions!